src/metabase/models/permissions_group_membership.clj

```diff
@@ -26,10 +26,20 @@
       (throw (ex-info (tru "You cannot add or remove users to/from the ''All Users'' group.")
                {:status-code 400})))))
 
-(defn- check-not-last-admin []
-  (when (<= (db/count PermissionsGroupMembership
-              :group_id (:id (perms-group/admin)))
-            1)
+(defn admin-count
+  "The current number of non-archived admins (superusers)"
```

Comment:
👍  for optimizing this.

Also, docstring should end with a punctuation https://guide.clojure.style/#docstring-grammar
```suggestion
  "The current number of non-archived admins (superuser)."
```